lms/djangoapps/bulk_email/migrations/0004_seed_aws_flags.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os
+from django.db import migrations, models
+
+def forwards_func(apps, schema_editor):
+    if os.environ["DJANGO_SETTINGS_MODULE"] == "lms.envs.aws":
```

Comment:
After speaking with @max and @feanil about this some more, we're not going to be including a seed value migration. This means that, until a value is added to the database, the `ENABLE_BULK_EMAIL` flag is effectively false.

From here, we have two possible routes of action:
1. Proceed with this PR as it exists, and enable the feature in the admin console after deploying. This will result in a short outage of the feature, which we'd need to communicate to partners as able.
2. Split this PR up:
  - Deploy 1: add the configuration model, but continue to use the pre-existing setting to actually control the feature
  - Post-deploy 1: Add a "bulk email is enabled" value to the configuration model table in admin console
  - Deploy 2: switch control of the feature flag over to the new model

Option 1 will be simpler, but will result in having the bulk email feature down for a short time during deployment. Option 2 is more complicated and will take longer, but will keep the feature working correctly without any downtime.

I'm leaning slightly towards option 1, but let's discuss @cahrens @dianakhuang